site/docs/pages/wallet/wallet.mdx

```diff
@@ -319,96 +319,6 @@ import { color } from '@coinbase/onchainkit/theme';
   </Wallet>
 </WalletComponents>
 
-## Using Wallet Modal
```

Comment:
i'm currently linking here in the walletisland doc, can you change that link to the new modal page you've created?